### PR TITLE
Finished Problems 31 through 34.

### PR DESCRIPTION
**Pull request by ikhat**
Problem 31 already had some solutions written in, so I just edited them a bit. 

33 and 34 were short so I did them both in one update. 

My solutions to 34 may be a bit too wordy for you, but I hope not.

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -2353,56 +2353,70 @@ \section*{Subspaces and Bases}
...
+				$\Set{\vec b,\vec c}$ is a basis for $Q$, and so its dimension is 2. 
+			\end{solution}
 		\item Is $P\cap Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				Yes. $\Set{\vec b}$ is a basis for $P\cap Q$, and so its dimension is 1. 
+			\end{solution}
 		\item Is $P\cup Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				No. For example $\vec a$ and $\vec c$ are both in $P\cup Q$, but 
+				$\vec a+\vec c \notin P \cup Q$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Justify this.

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -2437,22 +2451,65 @@ \section*{Matrices}
...
+				the columns of $A$, make the vector $\vec b$. In other words:
+				\[
+					x_0 \mat{1\\3} + y_0 \mat{2\\3} = \mat{-2\\-1}.
+				\]
+			\end{solution}
 		\item Let $\mat{x_0\\y_0}$ be a solution to $A\vec x=\vec b$. Explain what
 			$x_0$ and $y_0$ mean in terms of \emph{intersecting lines} (hint: think
 			about systems of equations).
+			\begin{solution}
+				The lines represented by the equations in the system above intersect
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Repeat the equations of the lines. The "system above" is a bit too vague.

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -2437,22 +2451,65 @@ \section*{Matrices}
...
 	\end{parts}
 
 	\question
 	Let $\vec u=\mat{1\\2\\3}$, $\vec v=\mat{4\\5\\6}$, $\vec w=\mat{7\\8\\9}$.
 	\begin{parts}
-		\item How could you determine if $\{\vec u,\vec v,\vec w\}$ was a linearly
+		\item How could you determine if $\Set{\vec u,\vec v,\vec w}$ was a linearly
 			independent set?
+			\begin{solution}
+				The set is is linearly independent if and only if the only linear
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Too much with a repeated `if` and a repeated `only`. Can you rephrase?

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -2353,56 +2353,70 @@ \section*{Subspaces and Bases}
...
+			\begin{solution}
+				$\Set{\vec a,\vec b}$ is a basis for $P$, and so its dimension is 2. 
+			\end{solution}
 		\item Give a basis for and the dimension of $Q$.
+			\begin{solution}
+				$\Set{\vec b,\vec c}$ is a basis for $Q$, and so its dimension is 2. 
+			\end{solution}
 		\item Is $P\cap Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				Yes. $\Set{\vec b}$ is a basis for $P\cap Q$, and so its dimension is 1.
```
[the remark this answers is not in the document]
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Sure. I'm writing out an explanation that essentially proves that the intersection of any two subspaces is a subspace. Would you prefer I do something more specific to this case? Like I could just prove that P \cap Q = span{b} and conclude everything from that.

[the remark this answers is not in the document]
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: I'm kind of doing both, as you'll see. I expect you'll want me to keep only one half of this. I'll wait for your word.

File: problemsets-mat223/linearalbegra.tex [hunk 5]
```diff
@@ -2353,56 +2353,98 @@ \section*{Subspaces and Bases}
...
+				No. For example $\vec a$ and $\vec c$ are both in $P\cup Q$, but 
+				$\vec a+\vec c \notin P\cup Q$. 
+
+				$\vec a+\vec c \notin P$ since if it was, we would also have
+				$(\vec a+\vec c)-\vec a=\vec c\in P$. We know this is impossible
+				since the vectors $\vec a,\vec b,\vec c$ are linearly independent,
+				and so $\vec c$ does not equal a linear combination of $\vec a$
+				and $\vec b$. 
+
+				An exactly analogous argument shows that $\vec a+\vec c \notin Q$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: -exactly

and spell out the definition of union. An alternative argument is:

I'd like something like: "A vector is in P\cup Q if it is in P or Q. v is in P if the equation $v=ta+sb$ is consistent and v is in Q if the equation $v=tb+sc$ is consistent. Since neither of these equations is consistent, v cannot be in P or Q, and is therefore not in P cup Q.

File: problemsets-mat223/linearalbegra.tex (hunk 4 above)
[the remark this answers is not in the document]
Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: You can just say two non-parallel planes intersect in a line. Since c and 0 are common to both planes, the line must contain c and 0 and is therefore span{c}

[the remark this answers is not in the document]
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: That's much easier.

File: problemsets-mat223/linearalbegra.tex (hunk 5 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: I'm spelling out the definition of union.

File: problemsets-mat223/linearalbegra.tex [hunk 6]
```diff
@@ -2353,56 +2353,87 @@ \section*{Subspaces and Bases}
...
+			\begin{solution}
+				$\Set{\vec b,\vec c}$ is a basis for $Q$, and so its dimension is 2. 
+			\end{solution}
 		\item Is $P\cap Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				Yes. $\Set{\vec b}$ is a basis for $P\cap Q$, and so its dimension is 1. 
+
+				$P$ and $Q$ are both planes and are not parallel (since 
+				$\vec a,\vec b,\vec c$ are linearly independent). The intersection
+				of any two non-parallel	planes intersect in a line.
```
[the remark this answers is not in the document]
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Sorry about that.
